skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -88,6 +90,16 @@ actual open class SkiaLayer internal constructor(
         add(backedLayer)
         backedLayer.addHierarchyListener {
             if (it.changeFlags and HierarchyEvent.SHOWING_CHANGED.toLong() != 0L) {
+                if (
```

Comment:
Is there a reason why it is in SHOWING_CHANGED handler, and not in `init {}` block? If we can avoid dynamically adding UI elements, and construct all swing UI in the constructor, we simplify the code.

Reply:
Because in `init {}` block we still do not know about actual `renderApi` but `SHOWING_CHANGED` also not the good place for that: replaced it inside `findNextWorkingRenderApi` method.